sdk/keyvault/azure-security-keyvault-keys/inc/azure/keyvault/keys/key_type.hpp

```diff
@@ -12,19 +13,48 @@
 
 namespace Azure { namespace Security { namespace KeyVault { namespace Keys {
 
-  enum class KeyTypeEnum
+  /**
+   * @brief The JsonWebKey types.
+   *
+   */
+  enum class JsonWebKeyType
```

Comment:
Is there a reason this isn't `KeyType` or `KeyTypeEnum`?  Exposing that the type is json binds the underlying data to be json.

cc: @JeffreyRichter

Reply:
> Exposing that the type is json

This is not true. The entire type is JsonWebKey (jwk) https://tools.ietf.org/html/rfc7517 . That's actually how it is called in the swagger spec.  .Net decided to renamed it to KeyType only.

We can't use KeyType because that's the name we want to give to a class property `KeyVaultKey.KeyType`  and it can't be named the same as how you named an enum type.

It used to be called KeyTypeEnum, but I never liked.  We don't add the `enum` postfix for any other enum in the code.